guides/common/modules/con_requirements-for-installation-in-an-ipv6-network.adoc

```diff
@@ -19,4 +19,4 @@ ifeval::["{build}" != "satellite"]
 Note that this requirement is for Katello users only.
 endif::[]
 
-* You must configure {Project} to use this IPv4 HTTP proxy server as the default proxy. For more information, see https://access.redhat.com/documentation/en-us/red_hat_satellite/6.8-beta/html/installing_satellite_server_from_a_connected_network/performing-additional-configuration#adding-a-default-http-proxy_satellite[Adding a Default HTTP Proxy to Satellite].
+* You must configure {Project} to use this IPv4 HTTP proxy server as the default proxy. For more information, see https://access.redhat.com/documentation/en-us/red_hat_satellite/6.8/html/installing_satellite_server_from_a_connected_network/performing-additional-configuration#adding-a-default-http-proxy_satellite[Adding a Default HTTP Proxy to Satellite].
```

Comment:
Why is there a hard 6.8 here and not {ProductVersion} variable? 
This is going to come back to haunt come the next release.